hbase-common/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKConfig.java

```diff
@@ -330,4 +341,27 @@ public static String getClientZKQuorumServersString(Configuration conf) {
     final String[] serverHosts = StringUtils.getStrings(clientQuromServers);
     return buildZKQuorumServerString(serverHosts, clientZkClientPort);
   }
+
+  private static void setZooKeeperClientSystemProperties(String prefix, Configuration conf) {
```

Comment:
@anmolnar Would you like to change the name of this method before merging? Or you prefer the original one?

Reply:
@wchevreuil @Apache9 Sorry, I couldn't look here over the weekend. Since the patch already has everything to submit I wouldn't make any change it for now. There's chance to have more ZK properties to add here which are not TLS related, so hopefully not a big deal.